src/machines/workflow_machines.rs

```diff
@@ -595,6 +607,15 @@ impl WorkflowMachines {
         Ok(())
     }
 
+    fn get_machine_key(&mut self, id: &CommandID) -> Result<MachineKey> {
+        Ok(*self.id_to_machine.get(id).ok_or_else(|| {
+            WFMachinesError::MissingAssociatedMachine(format!(
+                "Missing associated machine for cancelling timer {:?}",
```

Comment:
Text implies this can only be called in the context of timers but that's not true any more

Reply:
Done